pkgs/development/haskell-modules/configuration-ghc-9.2.x.nix

```diff
@@ -81,6 +81,7 @@ self: super: {
   extra = dontCheck super.extra;
 
   # Jailbreaks & Version Updates
+  aeson-diff = doJailbreak super.aeson-diff;
```

Comment:
We generally try to add links directly in the code as comments, so that other people coming by know when things like `doJailbreak` can be removed.

Would you be able to add a comment here (and the other file as well), pointing to https://github.com/thsutton/aeson-diff/pull/58 and saying that this `doJailbreak` can be removed when that PR is released to hackage?

Reply:
@cdepillabout I have changed the base branch to `haskell-updates` and added the comment to the files accordingly. Can you take one final look? Thanks!